src/src/PluginStructs/P104_data_struct.cpp

```diff
@@ -1154,8 +1154,9 @@ void reverseStr(String& str) {
     uint16_t n = str.length();
```

Comment:
You only use it in the loop as `n/2`
So why not make it const and divide it by 2 in the `n` variable?

Reply:
Done